### PR TITLE
Unit test on the newchannel viewmodel,clearafter viewmodel and editchannelViewmodel

### PR DESCRIPTION
**Pull request by DestinyOgbonna**
https://user-images.githubusercontent.com/80477923/135724054-96bd16c8-862b-42be-b2ac-6bcb7a87fe54.mp4

### CODE REVIEW

**Review by YenumJay — DISMISSED** [review]
Output results in your console do not clearly show your test works.

File: lib/ui/view/channel/edit_channel/edit_channel_view_model.dart [hunk 1]
```diff
@@ -34,28 +34,29 @@ class EditChannelViewModel extends FormViewModel {
       snackbar.showCustomSnackBar(
         duration: const Duration(seconds: 3),
         variant: SnackbarType.failure,
-        message: fillAllFields,
+        message: 'Please fill all fields.',
       );
 
       return;
     }
     const channel_id = '613f70bd6173056af01b4aba';
```
Comment (pauleke65) [inline lib/ui/view/channel/edit_channel/edit_channel_view_model.dart:42]: Don't use hard coded values.
get the org id from local storage.
pass the channel id through the function parameters

File: test/view_models/editchannels_viewmodel_test.dart [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  group('NewChannelTest -', () {
+    setUp(() => registerServices());
+    tearDown(() => unregisterServices());
+
+    group('initialize -', () {
+      test('When called, check if snackbar service was called', () async {
+        final snackbarService = getAndRegisterSnackbarServiceMock();
+        final model = EditChannelViewModel();
+        model.editChannel();
+        verifyNever(snackbarService.showCustomSnackBar());
```
Comment (pauleke65) [inline test/view_models/editchannels_viewmodel_test.dart:16]: Don't use verifyNever, use verify instead

File: test/view_models/editchannels_viewmodel_test.dart [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+        // verify(channelService.hasUser);
```
Comment (pauleke65) [inline test/view_models/editchannels_viewmodel_test.dart:17]: remove commented test

**Review by pauleke65 — CHANGES_REQUESTED** [review]
Changes Requested

**Comment by DestinyOgbonna** [comment]
Wrong PR

**Pull request closed without merging**